### DISCUSSION TITLE
Problem with nuxt router

### DISCUSSION THREAD

**Discussion by Masalion326** [nuxt/nuxt#10015] [Questions (Nuxt 2)]
When I try to make a page transition using the Router, it transitions to the corresponding URL, but a blank page is returned. Then, when I reload the blank page at the destination, the expected page is displayed. How can I solve this problem?

The problem occurs when you click on the Table page in the Tab of the initial page of the repository below.
I have checked the behavior with `npm run dev`.

https://github.com/Masalion326/nuxt-sample

**Comment by Hitsudzy** [comment]
up. have a same problem

**Reply by Hitsudzy** [reply]
solved:

https://v3.nuxtjs.org/docs/directory-structure/pages/
`Unlike components, your pages must have a single root element to allow Nuxt to apply route transitions between pages.`
